app/components/images/imagesController.js

```diff
@@ -59,7 +59,28 @@ function ($scope, $state, Config, Image, ImageHelper, Messages, Pagination) {
     });
   };
 
-  $scope.removeAction = function () {
+  $scope.confirmRemovalAction = function (force) {
+    ModalService.confirm({
```

Comment:
I'd go farther when isolating the bootbox inside the ModalService. ModalService methods should be pretty abstract and should not know about the required fields. It should only expose a `confirmationModal` method with the things that can change inside a confimation modal (e.g. **title**, **message**, **confirmLabel**, **cancelLabel** and the success callback).

Something like `ModalService.confirmModal('Are you sure?', Forcing the removal of the image will remove the image even if it has multiple tags or if it is used by stopped containers.', 'Remove the image', 'Cancel'`, function () { ... })